bidscoin/bids.py

```diff
@@ -246,6 +246,26 @@ def _extattributes(self) -> dict:
             for key, val in attributes.items():
                 if key not in self._cache:
                     self._cache[key] = val
+        # check for Excel/spreadsheet files
```

Comment:
Reading excel attributes is already implemented in the pet2bids plugin, so this code is redundant